scripts/make-g-cloud-live.py

```diff
@@ -138,10 +141,20 @@ def make_draft_service_live(client, copy_document, draft_service, framework_slug
     copy_document = document_copier(DRAFT_BUCKET, DOCUMENTS_BUCKET, DRY_RUN)
 
     suppliers = find_suppliers_on_framework(client, FRAMEWORK_SLUG)
+    results = collections.Counter({'success': 0, 'fail': 0})
 
     for supplier in suppliers:
         print(u"Migrating drafts for supplier {} - {}".format(supplier['supplierId'], supplier['supplierName']))
         draft_services = get_submitted_drafts(client, FRAMEWORK_SLUG, supplier['supplierId'])
-
         for draft_service in draft_services:
-            make_draft_service_live(client, copy_document, draft_service, FRAMEWORK_SLUG, DRY_RUN)
+            try:
+                make_draft_service_live(client, copy_document, draft_service, FRAMEWORK_SLUG, DRY_RUN)
+                results.update({'success': 1})
```

Comment:
Will this not overwrite a later fail? (i.e. fail, followed by success, will here = success. But I think any fail should give an exit code of 1.)